index.js

```diff
@@ -28,9 +28,17 @@ class PluginXrpPaychan extends PluginBtp {
     this._secret = opts.secret
     this._address = opts.address || deriveAddress(deriveKeypair(this._secret).publicKey)
 
+    if (typeof opts.currencyScale !== 'number' && opts.currencyScale !== undefined) {
```

Comment:
👍 

Remember to validate `opts.currencyScale` the same way in `-asym-server` and `-asym-client`